src/firebaseConfig.ts

```diff
@@ -72,18 +69,25 @@ type HostingBase = {
   };
 };
 
-type HostingSingle = HostingBase & {
+export type HostingSingle = HostingBase & {
   site?: string;
   target?: string;
 } & Deployable;
 
-type HostingMultiple = (HostingBase &
+export type HostingMultiple = (HostingBase &
```

Comment:
Also, as TL, you have the power to wave your hands and ask me to remove this HostingMultiple vs HostingSingle nonsense since the new utilities return a HostingResolved[] instead. 🙃